crates/typst-library/src/compute/data.rs

```diff
@@ -93,6 +93,26 @@ impl From<Readable> for Bytes {
     }
 }
 
+/// Writes plain text to a file.
```

Comment:
Perhaps adding examples and a bit more of a description would be nice. Have a look at the [read](https://typst.app/docs/reference/data-loading/read/) documentation for an example.